src/msspreadsheets/spreadsheet_types.bal

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Workbook properties.
+# + path - The path to the workbook in Microsoft OneDrive
+# + workbookName - The name of the workbook file (ignoring .xlsx extension name)
+public type WorkbookProperties record {|
+    string path;
+    string workbookName;
+|};
+
+# Worksheet properties.
+# + path - The path to the workbook in Microsoft OneDrive
+# + workbookName - The name of the workbook file (ignoring .xlsx extension name)
+# + worksheetName - The worksheet name
+# + sheetId - The unique identifier of the worksheet. This value is auto generated by Microsoft Graph
+# + position - The position of the worksheet
+public type WorksheetProperties record {|
+    string path;
+    string workbookName;
+    string worksheetName;
+    string sheetId;
+    int position;
+|};
+
+# Table properties.
+# + path - The path to the Workbook in Microsoft OneDrive
+# + workbookName - The name of the workbook file (ignoring .xlsx extension name)
+# + sheetId - The unique identifier of the worksheet. This value is auto generated by Microsoft Graph
+# + worksheetName - The worksheet name
+# + tableID - The unique identifier of the table. This value is auto generated by Microsoft Graph
+# + address - The location where the table should be created. E.g., A4:D4
```

Comment:
```suggestion
# + address - the location where the table should be create (e.g., A4:D4)
```

Reply:
I think this has to be "the location where the table should be created (e.g., A4:D4)"